roles/ntp/tasks/main.yml

```diff
@@ -0,0 +1,64 @@
+#
+# MIT License
+#
+# (C) Copyright 2023 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+---
+- name: Set timezone
+  community.general.timezone:
+    name: "{{ timezone }}"
+    hwclock: UTC
+
+- name: Render chrony template
+  ansible.builtin.template:
+    src: chrony.conf.j2
+    dest: /etc/chrony.conf
+    owner: root
+    group: root
+    mode: '0644'
+  notify:
+    - 'ntp : Restart chrony'
+
+- name: Render cray template
+  ansible.builtin.template:
+    src: cray.conf.j2
+    dest: /etc/chrony.d/cray.conf
+    owner: root
+    group: root
+    mode: '0644'
+  notify:
+    - 'ntp : Restart chrony'
+
+- name: Run handlers
+  ansible.builtin.meta: flush_handlers
+
+- name: Wait for time sync
+  ansible.builtin.wait_for:
+    timeout: 10
```

Comment:
There is an actual `chronyc waitsync` command if you wanted instead of an arbitrary timeout

Reply:
Oh, right on. I do!